src/ess/spectroscopy/indirect/workflow.py

```diff
@@ -635,16 +714,16 @@ def one_setting(
     kf_params, sample_detector_flight_time = find_sample_detector_flight_time(
         sample, analyzers, detector_positions
     )
-    sample_events = get_sample_events(triplet_events, sample_detector_flight_time)
+    events = get_sample_events(triplet_events, sample_detector_flight_time)
     ki_params, unwrapped_events, primary = get_unwrapped_events(
-        filename, names['source'], names['sample'], sample_events, names['focus']
+        filename, names['source'], names['sample'], events, names['focus']
     )
     ei, en, ef = get_energy_axes(ki_params, kf_params)
 
-    energy_events = sample_events.copy()
-    energy_events.bins.coords['energy_transfer'] = en.to(unit='meV')
-    energy_events.bins.coords['incident_energy'] = ei
-    energy_events.coords['final_energy'] = ef
+    # energy_events = sample_events.copy()
```

Comment:
Remove. Or make a shallow copy here to be sure.

Reply:
:+1: